src/app-router.js

```diff
@@ -212,7 +212,7 @@
     // update the references to the activeRoute and previousRoute. if you switch between routes quickly you may go to a
     // new route before the previous route's transition animation has completed. if that's the case we need to remove
     // the previous route's content before we replace the reference to the previous route.
-    if (router.previousRoute && router.previousRoute.transitionAnimationInProgress) {
+    if (router.previousRoute && (router.previousRoute.transitionAnimationInProgress || router.activeRoute.importInProgress)) {
```

Comment:
Trying to wrap my head around this scenario still. What combination of clicks/loads leads to this check?